src/producer.ts

```diff
@@ -9,4 +9,8 @@ export class Producer {
       new PublishRequest({ publisherId: this.publisherId, messages: [{ publishingId, message }] })
     )
   }
+
+  getLastPublishingId(stream: string, publisherRef: string) {
```

Comment:
Why don't you add `stream` and `publisherRef` to the ctor of Producer? These two value is related to producer

Reply:
added the values